setup/fio-setup-basic

```diff
@@ -175,6 +175,10 @@ __numa_cpu_nodes()
 	esac
 }
 
+is_mount_on_root() {
+	[[ $(df | grep $1 | awk '{print $6}') == '/' ]]
```

Comment:
i wonder whether this syntax can work in sh (which is not necessary bash). Also does awk need be installed through deps?

Reply:
no additional installation needed for awk but will use grep instead to maximize compatibility